Content.Server/GameObjects/Components/Power/PowerStorageComponent.cs

```diff
@@ -175,5 +176,12 @@ public void Examine(FormattedMessage message, bool inDetailsRange)
                 "[color=yellow]Charge: {0}J / {1}J ({2}%)\nRate: {3}W IN, {4}W OUT[/color]",
                 Math.Round(Charge, 2), Capacity, chargePercent, ChargeRate, DistributionRate));
         }
+
+        /// <inheritdoc />
+        public bool InteractUsing(InteractUsingEventArgs eventArgs)
+        {
+            if (Full) Owner.PopupMessage(eventArgs.User, "Capacitor at max charge");
+            return true;
+        }
```

Comment:
I would suggest, if you don't understand the purpose of the change, that asking on the issue is a good place to start. I don't think that this is what @ShadowCommander meant, because it doesn't make sense in my mind.

>I thought C# events didn't consume the "click"

This is not a C# event. This is our own interface, used by the click code. And returning `true` from it means "this interaction did something, no other interactions with other components should happen". So returning `true` prevents other components from being interacted with.